module/zfs/zcp_get.c

```diff
@@ -39,7 +39,7 @@
 #include <sys/zvol.h>
 
 #ifdef _KERNEL
-#include <sys/zfs_vfsops.h>
+#include <sys/zfs_quotas.h>
```

Comment:
This probably no longer need to be kernel only now that you've split it out in to its own source file.

Reply:
The quotas header includes the vfsops header, maybe I should leave the vfsops include here too since this file does use its types.

Reply:
Yes, leaving both I think would be clearer.